From a30dbd6a7985aa179a67aacdbb3d870e4eb1134f Mon Sep 17 00:00:00 2001
From: Andreas Schneider <asn@samba.org>
Date: Wed, 25 May 2016 13:39:29 +0200
Subject: [PATCH 1088/1096] s3-smbspool: Log to stderr

This way we will be able to see the log in the cups logs and are able to
debug issues.

https://bugzilla.samba.org/show_bug.cgi?id=11935

Signed-off-by: Andreas Schneider <asn@samba.org>
Reviewed-by: Michael Adam <obnox@samba.org>
(cherry picked from commit e59e0a5481196b1ddda0393306c514b8c32d6ea0)
(cherry picked from commit 0242e2aa5d6d27759b32d857b9b37e5306e9fb9b)
---
 source3/client/smbspool.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source3/client/smbspool.c b/source3/client/smbspool.c
index e381822..9dbb233 100644
--- a/source3/client/smbspool.c
+++ b/source3/client/smbspool.c
@@ -250,7 +250,7 @@ main(int argc,			/* I - Number of command-line arguments */
          * Setup the SAMBA server state...
          */
 
-	setup_logging("smbspool", DEBUG_STDOUT);
+	setup_logging("smbspool", DEBUG_STDERR);
 
 	smb_init_locale();
 
-- 
2.9.0

